indico/modules/rb/client/js/components/ClipboardButton.jsx

```diff
@@ -0,0 +1,58 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import {Popup, Icon} from 'semantic-ui-react';
+import {Translate} from 'indico/react/i18n';
+
+export default class ClipboardButton extends React.Component {
```

Comment:
We don't really use classes for new components anymore since functional comments are usually more lightweight

Reply:
Also, this shouldn't be in RB since it's a generic component; i think in one of my older comments I mentioned the correct location (in indico/web/...)